src/uct/api/uct.h

```diff
@@ -490,26 +523,51 @@ struct uct_iface_attr {
  * @ref uct_iface_open. User has to initialize all fields of this structure.
  */
 struct uct_iface_params {
-    ucs_cpu_set_t            cpu_mask;    /**< Mask of CPUs to use for resources */
-    const char               *tl_name;    /**< Transport name */
-    const char               *dev_name;   /**< Device Name */
-    ucs_stats_node_t         *stats_root; /**< Root in the statistics tree.
-                                               Can be NULL. If non NULL, it will be
-                                               a root of @a uct_iface object in the
-                                               statistics tree. */
-    size_t                   rx_headroom; /**< How much bytes to reserve before
-                                               the receive segment.*/
-
-    void                     *err_handler_arg; /**< Custom argument of
-                                                   @a err_handler. */
-    uct_error_handler_t      err_handler;      /**< The callback to handle
-                                                    transport level error.*/
-
-    /* These callbacks are only relevant for HW Tag Matching */
-    void                     *eager_arg;
-    uct_tag_unexp_eager_cb_t eager_cb;    /**< Callback for tag matching unexpected eager messages */
-    void                     *rndv_arg;
-    uct_tag_unexp_rndv_cb_t  rndv_cb;     /**< Callback for tag matching unexpected rndv messages */
+    /** Mask of CPUs to use for resources */
+    ucs_cpu_set_t                                cpu_mask;
+    /** Interface opening mode bitmap. @ref uct_iface_open_mode */
```

Comment:
I think it sounds a bit better to say "open mode" rather than "opening mode", but just my opinion.